tests/test_moderation.py

```
@@ -35,59 +35,41 @@ def test_parse_time_edge_cases(self):
-             patch('cogs.moderation.validate_hierarchy'), \
-             patch('cogs.moderation.log_moderation_action'):
-            
-            mock_path.return_value.mkdir.return_value = None
-            mock_path.return_value.exists.return_value = False
-            mock_json_load.return_value = {}  # Return empty dict for warnings
-            
-            mod = Moderation(mock_bot)
-            result = mod._save_warnings()
-            
-            assert result is True
+        mod = Moderation(mock_bot)
```

Ruff (0.12.2)

Local variable `mod` is assigned to but never used

Remove assignment to unused variable `mod`

(F841)